docs/manual/transform_data.rst

```diff
@@ -0,0 +1,512 @@
+
+Datatable Functions can be applied to the columns::
+
+   >>> DT[:, dt.math.pow(f.floats, 3)]
+      |       C0
+      |  float64
+   -- + --------
+    0 |  1000
+    1 |  1520.88
+    2 |  1860.87
+    3 | -2197
+   [4 rows x 1 column]
+
+   >>> DT[:, dt.time.year(f.dates)]
+      | dates
+      | int32
+   -- + -----
+    0 |  2000
+    1 |  2010
+    2 |  2020
+    3 |    NA
+   [4 rows x 1 column]
+
+Functions can be applied across columns, row-wise and column-wise::
+
+   >>> DT[:, f['integers':'floats'].sum()]
+      | integers   floats
+      |    int64  float64
+   -- + --------  -------
+    0 |       10     20.8
+   [1 row x 2 columns]
+
+   >>> DT[:, f['integers':'floats'].rowsum()]
+      |      C0
+      | float64
+   -- + -------
+    0 |    11
+    1 |    13.5
+    2 |    15.3
+    3 |    -9
+   [4 rows x 1 column]
+
+Transformation of a column based on a condition is possible, via :func:`ifelse()`, which operates similarly to Python's `if-else` idiom::
+
+   >>> DT[:, ifelse(f.integers % 2 == 0, 'even', 'odd')]
+      | C0
+      | str32
+   -- + -----
+    0 | odd
+    1 | even
+    2 | odd
+    3 | even
+   [4 rows x 1 column]
+
+Iteration on a Frame
+--------------------
+Iterating through a :class:`Frame` allows access to the individual columns; each column is treated as a :class:`Frame`::
+
+   >>> [frame for frame in DT]
+
+      | dates
+      | date32
+   -- + ----------
+    0 | 2000-01-05
+    1 | 2010-11-23
+    2 | 2020-02-29
+    3 | NA
+   [4 rows x 1 column]
+
+      | integers
+      |    int32
+   -- + --------
+    0 |        1
+    1 |        2
+    2 |        3
+    3 |        4
+   [4 rows x 1 column]
+
+      |  floats
+      | float64
+   -- + -------
+    0 |    10
+    1 |    11.5
+    2 |    12.3
+    3 |   -13
+   [4 rows x 1 column]
+
+      | strings
+      | str32
+   -- + -------
+    0 | A
+    1 | B
+    2 | NA
+    3 | D
+   [4 rows x 1 column]
+
+With iteration, different operations can be applied to different columns::
+
+   >>> outcome = [frame.mean() if frame.type.is_numeric else frame[0, :] for frame in DT]
+   >>> outcome
+      | dates
+      | date32
+   -- + ----------
+    0 | 2000-01-05
+   [1 row x 1 column]
+
+      | integers
+      |  float64
+   -- + --------
+    0 |      2.5
+   [1 row x 1 column]
+
+      |  floats
+      | float64
+   -- + -------
+    0 |     5.2
+   [1 row x 1 column]
+
+      | strings
+      | str32
+   -- + -------
+    0 | A
+   [1 row x 1 column]
+
+
+   >>> DT[:, outcome] # or dt.cbind(outcome)
+      | dates       integers   floats  strings
+      | date32       float64  float64  str32
+   -- + ----------  --------  -------  -------
+    0 | 2000-01-05       2.5      5.2  A
+   [1 row x 4 columns]
+
+
+Sorting a Frame
+---------------
+A :class:`Frame` can be sorted via the :func:`sort()` function, or the :meth:`datatable.Frame.sort` method::
```

Comment:
We normally spell it as a "Frame" when we're talking about the class (smth like "Frame's methods"), and as a "frame" when we're talking about an object (smth like "a frame can be sorted"). As such, here I would propose to spell it as "frame" without hyperlink, because it is not the class but any object that can be sorted.